### ISSUE TITLE
Version 1.7.4 in the JS file

### ISSUE THREAD

**Issue by andreapernici** [NickPiscitelli/Glider.js#243]
I suppose the version in the JS file should be updated. https://github.com/NickPiscitelli/Glider.js/blob/master/glider.js

**Comment by NickPiscitelli** [comment]
Yes, definitely. I will update the build system to handle the file as well. Thank you.
